docs/concepts/workers.md

```diff
@@ -4,4 +4,52 @@ title: Workers
 sidebar_label: Workers
 ---
 
-A Worker is a service that hosts the [Workflow](/docs/concepts/workflows) and [Activity](/docs/concepts/activities) implementations.
+A Worker is a service that does the following:
+
+- Hosts executable [Workflow](/docs/concepts/workflows) and [Activity](/docs/concepts/activities) code.
+- Listens to [Task Queues](/docs/concepts/task-queues) via long polling.
+
+Workers must have access to any resources needed to execute the actions defined in Activities, such as the following:
+
+- Network access for external API calls.
+- Credentials for infrastructure provisioning.
+- Specialized GPUs for machine learning utilities.
+
+
+:::note
+
+If you need to process work sequentially on the same machine, the Go SDK also offers a [Sessions API](https://docs.temporal.io/docs/go/sessions/).
+
+:::
+
+See example Worker code for:
+
+- [Go SDK](/docs/go/workers)
+- [Java SDK](/docs/java/run-your-first-app-tutorial/#the-worker)
+- [PHP SDK](/docs/php/workers)
+- [Node SDK](/docs/node/hello-world/#worker)
+
+## Workers are external to the Temporal Server
+
+In our tutorials, we show you how to run both the Temporal Server and one Worker on the same machine for local development.
```

Comment:
which tutorials? (maybe we can add a link)

Reply:
hard to link, as there are different tutorials for every SDK